site/app/templates/admin/LibraryManager.twig

```diff
@@ -0,0 +1,43 @@
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+
+<div class="content">
+    <header>
+        <div>
+            <h1>Library Manager</h1>
+        </div>
+        <div>
+            <p>{{ text }}</p>
+        </div>
+    </header>
+    <br/>
+    <div class="tab-bar-wrapper">
+        <a class="nav-bar normal-btn active-btn" id="page_0_nav" href="javascript:onChangeNavTab(0);">Upload</a>
+        <a class="nav-bar normal-btn" id="page_1_nav" href="javascript:onChangeNavTab(1);">File Path</a>
+        <a class="nav-bar normal-btn" id="page_2_nav" href="javascript:onChangeNavTab(2);">Git</a>
+    </div>
+    <br/>
+    <br/>
+    <div class="modal-body">
```

Comment:
Just curious where/whether this class is defined somewhere, I couldn't find it. Are we planning on turning the library manager view into an actual modal instead of separate route?

Reply:
That was part of something I copied from AdminGradeableBase.twig and forgot about. It doesn't look like it is defined there either. I think the current plan is for it to be a page.